abstract-cz.tex

```diff
@@ -10,5 +10,6 @@
 \input{todos}
 
 \begin{document}
+\thispagestyle{empty}
```

Comment:
this is total nitpicking but I guess we could turn it off for all pages, given someone writes an abstract that spans 2 pages?

```suggestion
\pagenumbering{gobble}
```

Reply:
Good point 👍